utils/fix_cmocka_xml.py

```diff
@@ -49,7 +48,7 @@
         print("Skipping %s, it's a directory" % file)
         continue
     if file != "":
-        file_handle =  open('{0}'.format(file), "r+")
+        file_handle = open('{0}'.format(file), "r+")
```

Comment:
(pylint-consider-using-with) Consider using 'with' for resource-allocating operations